extensions/ros2/launch

```diff
@@ -39,6 +39,9 @@ if [ -d "$SNAP/opt/ros/underlay_ws/opt/ros/$ROS_DISTRO" ] || [ -d "$SNAP/opt/ros
 fi
 
 source_with_prefix "$SNAP/opt/ros/$ROS_DISTRO"
```

Comment:
could it ever happen that there is no global setup.bash?

Reply:
No, it shouldn't and thus this is left as possibly error trigerring.